File: BubbleSort.ts
const bubble_sort = (array: number[]) => {
  for (let i = 0; i <= array.length - 1; i++) {
    for (let j = 0; j <= array.length - 1 - i; j++) {
      if (array[j] > array[j + 1]) {
        const x = array[j];
        array[j] = array[j + 1];
        array[j + 1] = x;
      }
    }
  }

  return array;
};

bubble_sort([9, 1, 3, 7, 5, 2]);

File: BubbleSort.py
def bubble_sort(array):
    iterateLength = len(array) - 1

    while iterateLength > 0:
        index = 0
        for _ in range(iterateLength):
            if array[index] > array[index + 1]:
                x = array[index]
                array[index] = array[index + 1]
                array[index + 1] = x
            index += 1
        iterateLength -= 1

    return array


bubble_sort([9, 2, 7, 5, 1])
bubble_sort([11, 5, 2, 9, 8, 55, 1])
